### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by varantsou**

### CODE REVIEW

File: task1/js/app.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    'strong': 0,
+    'mad': 0,
+    'small': 0,
+    'dead': 0
+};
+
+for (let i = 0; i < zombiesData.length; i++) {
+    if (zombiesData[i] !== undefined) {
+        if (zombiesData[i] < 1 || isNaN(zombiesData[i])) {
+            statistics.dead += 1;
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: statistics.dead++

File: task1/js/app.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        } else if (zombiesData[i] >= 1 && zombiesData[i] < 11) {
+            statistics.small += 1;
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: same here

File: task1/js/app.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        } else if (zombiesData[i] >= 11 && zombiesData[i] < 21) {
+            statistics.mad += 1;
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: same here

File: task1/js/app.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        } else if (zombiesData[i] >= 21) {
+            statistics.strong += 1;
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: same here

File: task1/js/app.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        statistics.total += 1;
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: same here

File: task1/js/app.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    }
+}
+
+const totalElement = document.querySelector('.total-zombies');
+const deadElement = document.querySelector('.dead-zombies');
+const smallElement = document.querySelector('.small-zombies');
+const madElement = document.querySelector('.mad-zombies');
+const strongElement = document.querySelector('.strong-zombies');
+
+const printInConsole = function() {
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: what's the point of this function declaration?

File: task1/js/app.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    console.log(totalElement.innerText, statistics.total);
+    console.log(deadElement.innerText, statistics.dead);
+    console.log(smallElement.innerText, statistics.small);
+    console.log(madElement.innerText, statistics.mad);
+    console.log(strongElement.innerText, statistics.strong);
+};
+
+const changeHTMLStatistics = function() {
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: what's the point of this function declaration?

**Review by ExSamaelSolbeg — COMMENTED** [review]
mostly ok
just minor refactoring

File: task1/js/app.js (hunk 1 above)
Comment (varantsou) [reply task1/js/app.js]: done

File: task1/js/app.js (hunk 2 above)
Comment (varantsou) [reply task1/js/app.js]: done

File: task1/js/app.js (hunk 3 above)
Comment (varantsou) [reply task1/js/app.js]: done

File: task1/js/app.js (hunk 4 above)
Comment (varantsou) [reply task1/js/app.js]: done

File: task1/js/app.js (hunk 5 above)
Comment (varantsou) [reply task1/js/app.js]: done

File: task1/js/app.js (hunk 6 above)
Comment (varantsou) [reply task1/js/app.js]: removed

File: task1/js/app.js (hunk 7 above)
Comment (varantsou) [reply task1/js/app.js]: removed

**Review by ExSamaelSolbeg — DISMISSED** [review]

File: task1/js/app.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    'total': 0,
+    'strong': 0,
+    'mad': 0,
+    'small': 0,
+    'dead': 0
+};
+
+for (let i = 0; i < zombiesData.length; i++) {
+    if (zombiesData[i] !== undefined) {
+        if (zombiesData[i] < 1 || +zombiesData[i] !== zombiesData[i]) {
```
Comment (ExSamaelSolbeg) [inline task1/js/app.js]: if (zombiesData[i] < 1 || +zombiesData[i] !== zombiesData[i]) {
wtf?

**Review by ExSamaelSolbeg — CHANGES_REQUESTED** [review]

File: task1/js/app.js (hunk 8 above)
Comment (varantsou) [reply task1/js/app.js]: remove

**Review by ExSamaelSolbeg — APPROVED** [review]

**Pull request merged**